app/models/manageiq/providers/azure/cloud_manager/event_catcher/stream.rb

```diff
@@ -1,4 +1,18 @@
 class ManageIQ::Providers::Azure::CloudManager::EventCatcher::Stream
+  SELECT_FIELDS = %w[
+    authorization
+    correlationId
+    description
+    eventDataId
+    eventName
+    eventTimestamp
+    operationName
+    resourceGroupName
+    resourceProviderName
+    resourceId
+    resourceType
+  ].join(',')
```

Comment:
I would add `.freeze` here, as it's a constant.

Reply:
Done!